src/Quaternions.jl

```diff
@@ -8,9 +8,22 @@ module Quaternions
   import Base: rand, randn
   import LinearAlgebra: lyap, norm, normalize, sylvester
   using Random
+  using DualNumbers
 
   Base.@irrational INV_SQRT_EIGHT 0.3535533905932737622004 sqrt(big(0.125))
 
+  """
+      isunit(x)
```

Comment:
This naming is most intuitive (we say a "unit quaternion"), but `issign` would be more consistent with the definition that `x == sign(x)`.